crates/holochain/src/core/sys_validate/error.rs

```diff
@@ -147,6 +147,8 @@ pub enum ValidationOutcome {
     WrongDna(DnaHash, DnaHash),
     #[error("Update original EntryType: {0:?} doesn't match new EntryType {1:?}")]
     UpdateTypeMismatch(EntryType, EntryType),
+    #[error("Update original {0:?} doesn't match the {1:?} in the update")]
```

Comment:
This probably prints as `Update original EntryHash { ... } doesn't match the EntryHash { ... } in the update`. You could update the other errors accordingly to match.

Reply:
Yes, it absolutely does, I've updated several of the others